boost-maven/boost-runtimes/runtime-openliberty/src/main/java/org/microshed/boost/runtimes/openliberty/LibertyRuntime.java

```diff
@@ -267,13 +277,22 @@ private void installMissingFeatures() throws MojoExecutionException {
      */
     private void installApp(String installAppPackagesVal) throws MojoExecutionException {
 
-        Element deployPackages = element(name("deployPackages"), installAppPackagesVal);
-        Element serverNameElement = element(name("serverName"), serverName);
+        // Since the app is installed as a loose app, there is no need to
+        // re-install if
+        // the server is already running. If we do, the app will be deployed to
+        // dropins
+        // which is not what we want.
+        if (!isServerRunning()) {
```

Comment:
I"m confused if we're doing this as a workaround to the ci.maven issue or permanently?